worker/transcribee_worker/worker.py

```diff
@@ -23,14 +23,24 @@
 from transcribee_worker.whisper_transcribe import transcribe_clean
 
 
-def ensure_timing_invariant(doc: EditorDocument):
+def ensure_atom_invariants(doc: EditorDocument):
```

Comment:
is this used at all?

Reply:
Not at the moment, I wasn't sure where the right place for this would be. But i'd keep it as a reminder to add this somewhere soon